operator/src/main/java/oracle/kubernetes/weblogic/domain/model/IntrospectorJobEnvVars.java

```diff
@@ -55,6 +55,71 @@
    */
   public static final String WDT_DOMAIN_TYPE = "WDT_DOMAIN_TYPE";
 
+  /**
+   * MII Use Online Update.
+   */
+  public static final String MII_USE_ONLINE_UPDATE = "MII_USE_ONLINE_UPDATE";
+
+  /**
+   * MII MII dynamic update on non-dynamic changes cancelUpdate.
+   */
+  public static final String MII_CANCEL_CHANGES_IFRESTART_REQUIRED = "MII_CANCEL_CHANGES_IFRESTART_REQ";
+
+  /**
+   * MII dynamic update on non-dynamic changes CommitUpdateAndRoll.
+   */
+  public static final String MII_COMMIT_AND_ROLL = "MII_UPD_COMMIT_AND_ROLL";
+
+  /**
+   * MII cancel changes on non-dynamic changes CommitUpdateOnly.
+   */
+  public static final String MII_COMMIT_ONLY = "MII_UPD_COMMIT_ONLY";
+
+  /**
+   * WDT CONNECT TIMEOUT.
+   */
+
+  public static final String WDT_CONNECT_TIMEOUT = "wdt_connect_timeout";
```

Comment:
It's a nice convention to upper-case all env vars as you did for the ^^^ env vars - so please do the same for the timeouts.  Also, it's nice that you started a good many of the ^^^ model-in-image related env vars with "MII_" since that clearly delineates their purpose -- so can you do the same for the timeouts?

Reply:
I assume the timeouts are all 'update' related?  So maybe they should all start with MII_UPD_...